<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="G:\CP_lv4\PROG2070_Programming Software Quality\a04\A04UnitTest\A04UnitTest\bin\Debug\A04UnitTest.dll" total="5" errors="0" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2017-04-20" time="15:39:24">
  <environment nunit-version="2.6.4.14350" clr-version="2.0.50727.5485" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="C:\Program Files (x86)\NUnit 2.6.4\bin" machine-name="2A211-D03" user="ycho5551" user-domain="CONESTOGAC" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="G:\CP_lv4\PROG2070_Programming Software Quality\a04\A04UnitTest\A04UnitTest\bin\Debug\A04UnitTest.dll" executed="True" result="Success" success="True" time="17.081" asserts="0">
    <results>
      <test-suite type="Namespace" name="A04UnitTest" executed="True" result="Success" success="True" time="17.080" asserts="0">
        <results>
          <test-suite type="TestFixture" name="AllFieldErrorCase" executed="True" result="Success" success="True" time="3.190" asserts="0">
            <results>
              <test-case name="A04UnitTest.AllFieldErrorCase.AllFieldErrorCaseTest" executed="True" result="Success" success="True" time="3.188" asserts="2" />
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="EmailAndMandatoryErrorCase" executed="True" result="Success" success="True" time="3.319" asserts="0">
            <results>
              <test-case name="A04UnitTest.EmailAndMandatoryErrorCase.EmailAndMandatoryErrorCaseTest" executed="True" result="Success" success="True" time="3.317" asserts="2" />
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="EmailErrorCaseTest" executed="True" result="Success" success="True" time="3.613" asserts="0">
            <results>
              <test-case name="A04UnitTest.EmailErrorCaseTest.EmailErrorCaseTestTest" executed="True" result="Success" success="True" time="3.611" asserts="2" />
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="NormalInputCaseTest" executed="True" result="Success" success="True" time="3.458" asserts="0">
            <results>
              <test-case name="A04UnitTest.NormalInputCaseTest.NormalInputCaseTestTest" executed="True" result="Success" success="True" time="3.457" asserts="0" />
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="NormalInputSuccessCase" executed="True" result="Success" success="True" time="3.460" asserts="0">
            <results>
              <test-case name="A04UnitTest.NormalInputSuccessCase.NormalInputSuccessCaseTest" executed="True" result="Success" success="True" time="3.458" asserts="1" />
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>